DashKitUI/View Models/DeactivatePodViewModel.swift

```diff
@@ -147,13 +147,8 @@ enum DeactivationError : LocalizedError {
     
     var recoverySuggestion: String? {
         switch self {
-        case .podCommError(let error):
-            switch error {
-            case .bleCommunicationError, .notConnected:
-                    return String(format: LocalizedString("If this problem persists, tap Discard Pod. You can then activate a new Pod", comment: "Format string for ble communication error recovery suggestion during pairing. (1: device model name)"), UIDevice.current.model)
-            default:
-                return error.recoverySuggestion
-            }
+        case .podCommError:
+            return String(format: LocalizedString("If this problem persists, tap Discard Pod. You can then activate a new Pod", comment: "Format string for recovery suggestion during deactivate pod. (1: device model name)"), UIDevice.current.model)
```

Comment:
[nit] Am I overlooking were the device model is used in the string format?

Reply:
Ah, nope. :) Will clean that up.